SVG.flood/layer/image.svg.flt

```diff
@@ -10,8 +10,7 @@ var y = getImageY(layer: layer)
 <g id="<%= id %>-layer-translate" transform="<%= transform(of: layer) %>">
   <svg x="0" y="0" preserveAspectRatio="<%= aspectRatio %>" viewBox="<%= x %> <%= y %> <%= viewWidth %> <%= viewHeight %>" width="<%= width %>" height="<%= height %>">
     <use href="#<%= layer.image.name %>">
-    <% setAnimation(property: "image", layer: layer, timeline: timeline) %>	
-    <% animation(property: "contentMode", layer: layer, timeline: timeline, hrefID: nil, attributeType: nil) %>
+    <%= animation(property: "fillImage", layer: layer, timeline: timeline, hrefID: nil, attributeType: nil)%>
```

Comment:
This change worries me? This probably breaks image animations?

Reply:
This template is used for image Layers no?
But Fill images only apply to shapeLayers...